.github/workflows/ttnn-run-sweeps.yaml

```diff
@@ -12,40 +12,45 @@ on:
           - ALL SWEEPS (Nightly)
           - add
           - line_all_gather
-          - logical_and_
-          - eltwise.unary.relu.relu_interleaved
-          - eltwise.unary.gelu.gelu_interleaved
-          - eltwise.unary.cos.cos_interleaved
-          - eltwise.unary.sin.sin_interleaved
-          - eltwise.unary.clamp.clamp_interleaved
-          - eltwise.unary.clip.clip_interleaved
-          - eltwise.unary.cbrt.cbrt_interleaved
-          - eltwise.unary.rsub.rsub_interleaved
-          - eltwise.unary.rdiv.rdiv_interleaved
-          - eltwise.unary.frac.frac_interleaved
-          - eltwise.unary.ceil.ceil_interleaved
-          - eltwise.unary.trunc.trunc_interleaved
-          - eltwise.unary.floor.floor_interleaved
-          - eltwise.unary.clone.clone_interleaved
-          - eltwise.unary.exp.exp_interleaved
-          - eltwise.unary.exp2.exp2_interleaved
-          - eltwise.unary.expm1.expm1_interleaved
-          - eltwise.unary.tanh.tanh_interleaved
-          - eltwise.unary.sign.sign_interleaved
-          - eltwise.unary.rad2deg.rad2deg_interleaved
-          - eltwise.unary.deg2rad.deg2rad_interleaved
-          - eltwise.unary.relu6.relu6_interleaved
-          - eltwise.binary.subtract.subtract_interleaved
-          - eltwise.binary.multiply.multiply_interleaved
-          - eltwise.binary.div.div_interleaved
-          - eltwise.binary.div_no_nan.div_no_nan_interleaved
-          - eltwise.binary.logical_or.logical_or_interleaved
-          - eltwise.composite.binary.addalpha.addalpha_interleaved
-          - eltwise.composite.binary.subalpha.subalpha_interleaved
-          - eltwise.ternary.addcmul.addcmul_interleaved
-          - eltwise.ternary.addcdiv.addcdiv_interleaved
-          - eltwise.ternary.mac.mac_interleaved
-          - eltwise.ternary.where.where_interleaved
+          - eltwise.unary.relu.relu
```

Comment:
Is there a reason for all these eltwise files to be nested in folders? Are there plans to add more test modules in each one? If not can we make them top level?

Reply:
I understood that that is the plan. I will double check it.

Reply:
For some of these ops there is a plan to make another test modules, in next PR i'll make the test_modules which don't have more use cases to be top level.

Reply:
Thanks